accepted/005-node-authentication.md

```diff
@@ -0,0 +1,77 @@
+# Design / Feature
+
+
+This feature will allow an administrative user to authenticate with the node.
+
+#### Scope
+This document only covers node authorization. Application authorization is out
+of scope for this proposal.
+
+#### Policy Engine
+[Proposal 002](/accepted/002.md) provides the aurae daemon with an SSH
+authenticated user based upon SSH Certificates. The selected authentication
+provides identity and communication channel encryption.
+
+Authorization consumes the identity provided by Proposal 002. When the user
+connects, the user identity is authenticated directly from the connection. Upon
+initial connection, the identity is sent to an authorization plugin.
+
+The authorization engine receives a message with the identity, action, and
+parameters. The authorization engine returns a decision. Aurae Daemon is
+responsible for enforcing the decision.
+
+Authorization decisions are sent to the observability module.
+
+A simple default policy engine written in rust will be installed when no other
+authorization engine has been installed. Users may choose to implement their
+own policy engine by implementing the relevant trait.
+
+Authorization decisions default to deny.
+
+Not installing a policy engine causes all requests to be denied.
```

Comment:
We must must have "reasonable defaults" and opinions. Its a key part of the project.

Is this our reasonable default? Or should we give the user a default policy engine?